### DISCUSSION TITLE
How can I correct this mistake, please see screen shot

### DISCUSSION THREAD

**Discussion by mahalepoto** [community/community#21922] [New to GitHub]
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/1X/0aec6e9eee2d8b5c2359c81bc3f148b7fdd544f6.png" data-download-href="/uploads/short-url/1yDkltYRMsSp95z2sMofgu9wENE.png?dl=1" title="Screen Shot 2018-02-07 at 13.05.19.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/25729939/181067537-8232a337-45c9-4d69-a231-63019778682f.png" alt="Screen Shot 2018-02-07 at 13.05.19.png" data-base62-sha1="1yDkltYRMsSp95z2sMofgu9wENE" width="690" height="185" srcset="https://aws1.discourse-cdn.com/github/optimized/1X/0aec6e9eee2d8b5c2359c81bc3f148b7fdd544f6_2_690x185.png, https://aws1.discourse-cdn.com/github/optimized/1X/0aec6e9eee2d8b5c2359c81bc3f148b7fdd544f6_2_1035x277.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/1X/0aec6e9eee2d8b5c2359c81bc3f148b7fdd544f6_2_1380x370.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/1X/0aec6e9eee2d8b5c2359c81bc3f148b7fdd544f6_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Screen Shot 2018-02-07 at 13.05.19.png</span><span class="informations">2182×586 94.1 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>As you are able to see there is a long line of commands printed on my repo, dont know how it got there,just above PAM.cc, below branch:master. Can some body tell me how to edit or delete that.</p>
<p>Many Thanks</p>

**Comment by lee-dohm** [comment]
<p>What’s displayed there is the first line of the commit comment of the latest commit to the repository. You can submit a new commit (even an empty one) with a shorter first line to make it look better. From the command line in your local copy of your repository you could use:</p>
<pre><code>git commit --allow-empty -m "Better commit message"
git push
</code></pre>
<p>I hope that helps!</p>

**Comment by mahalepoto** [comment]
<p>Hi, </p>
<p>   thanks for reply…I tried your commands but did not work then I deleted the remote repo and .git hidden folder from local and tried again but again same thing happen. Im not sure what it is. Thanks any way.</p>

**Comment by lee-dohm** [comment]
<p>If the repository is public so you can share a link to it, I might be able to help diagnose further.</p>

**Comment by mahalepoto** [comment]
<p>Thanks for replying again. I deleted again and re-created remote repo but problem still there, dont know why, followed exact procedure showed when creating new remote repo on github, that weird comments still there, below is the link </p>
<p><a href="https://github.com/mahalepoto/Udacity-1st-Project-PAM" rel="nofollow noopener">https://github.com/mahalepoto/Udacity-1st-Project-PAM</a></p>
<p>cheers</p>

**Comment by lee-dohm** [comment, marked as answer]
<p>I found the problem, here’s the output from <code>git log</code> on your repository:</p>
<pre><code>Author: “mahalepotocd css/Udacity/PAM-Working/git initdefaults write com.apple.finder AppleShow AllFiles YESdefaults write com.apple.finder AppleShowAllFiles YEScd Users/mh/Desktop/CSS/Udacity/PAM-Working/filescd filescd /Users/mh/Desktop/CSS/Udacity/PAM-Working/exitgit config --global user.name “mahalepoto &lt;mahalepot&gt;
Date: Wed Feb 14 00:18:26 2018 +0000

    First Commit
</code></pre>
<p>Somehow you’ve told Git that your user name is:</p>
<pre><code>mahalepotocd css/Udacity/PAM-Working/git initdefaults write com.apple.finder AppleShow AllFiles YESdefaults write com.apple.finder AppleShowAllFiles YEScd Users/mh/Desktop/CSS/Udacity/PAM-Working/filescd filescd /Users/mh/Desktop/CSS/Udacity/PAM-Working/exitgit config --global user.name “mahalepoto &lt;mahalepot&gt;
</code></pre>
<p>If you go to the command line and type <code>git config --global user.name mahalepoto</code>, press <code>Enter</code>, and then start over, things should work better this time.</p>

**Comment by mahalepoto** [comment]
<p>It works fine now…Many thanks for your help.</p>
